distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -6,130 +6,149 @@
 
 image::navigator-icon.png[navigator icon]
 
-.Select the desired view from the *navigator* menu.
-image::navigator-menu-icon.png[navigator menu]
+.Select the desired view from the *Navigation* menu.
+image::navigator-menu-icon.png[navigation menu]
 
 === Workspaces in ${catalog-ui}
 
-Within ${catalog-ui}, workspaces are collections of settings, searches, and bookmarks that can be shared between users and stored to be accessed repeatedly.
+Within ${catalog-ui}, workspaces are collections of settings, searches, and bookmarks that can be shared between users and stored for repeated access.
 
 ==== Creating a Workspace in ${catalog-ui}
 
 Before searching in ${branding}, at least one workspace must be created.
-Use an existing template or create a blank workspace.
+
+.Start new workspace
+. From the *Workspaces* view, enter search terms into the *Start new workspace* search field and click the magnifying glass (image:magnifying-glass.png[magnifying glass icon]) icon. This will create a new workspace and perform a search based on the entered search terms.
+
+.Start new workspace
+image::start-new-workspace.png[start new workspace icon]
+
+.Using a template
+. From the *Workspaces* view, click on an existing template.
+. Change the workspace title by clicking on the temporary workspace title in the upper left corner and entering a new title.
+. Click the save (image:save-icon.png[save icon]) icon next to the workspace title in the upper left corner.
 
 .Default Workspace Templates
 image::workspace-templates.png[workspace templates]
 
-Local:: an example of a local search.
-Federated:: an example of a example of a search across all connected sources.
-Location:: an example of a geographically constrained search.
-Temporal:: an example of a time-range search.
+Blank:: A blank workspace with no default search.
+Local:: An example of a local search.
+Federated:: An example of a search across all federated sources and the local catalog.
+Location:: An example of a geographically constrained search.
+Temporal:: An example of a time-range search.
 
 ==== Configuring a Workspace in ${catalog-ui}
 
-Configure each workspace with queries and share options.
+Configure each workspace with searches and share options.
 
 .Adding searches
-. Select the default name of the workspace (*New Workspace* or template name) to change to a unique name.
-. If *Blank* workspace was selected, select *Add Search* to add the first search. Each workspace can have up to ten searches.
-.. Change the name of the search.
+. From the default *Workspaces* view, select the workspace to add a search to.
+. Click *Search ${branding} Intrigue* in the upper left corner, enter search terms, and click *Search* to add a search. This step can be repeated to add additional searches. Each workspace can have up to ten searches.
```

Comment:
❓ is 10 still the limit, is it configurable? Not sure there is any action there just a question. If it is configurable probably worth a mention. if not, then it is fine.

Reply:
Yes, 10 is still the limit and it is not configurable at this time.